### PR TITLE
Added GetVisible() to the MeshFeatureProcessor

### PR DESCRIPTION
**Pull request by kh-huawei**
## What does this PR do?
Added the interface function `bool GetVisible(const MeshHandle& meshHandle) const` for the MeshFeatureProcessor, since it was missing.
 
## How was this PR tested?

Use the function in code to find out the visibility state of a mesh.

### CODE REVIEW

**Review by gadams3 — APPROVED** [review]

File: Gems/Atom/Feature/Common/Code/Source/Mesh/MeshFeatureProcessor.cpp [hunk 1]
```diff
@@ -466,6 +466,16 @@ namespace AZ
...
+        bool MeshFeatureProcessor::GetVisible(const MeshHandle& meshHandle) const
+        {
+            if (meshHandle.IsValid())
+            {
+                return meshHandle->m_visible;
+            }
+            return false;
+        }
+
+
```
Comment (pollend) [inline Gems/Atom/Feature/Common/Code/Source/Mesh/MeshFeatureProcessor.cpp:478]: nit: extra carriage return

Comment (kh-huawei) [reply Gems/Atom/Feature/Common/Code/Source/Mesh/MeshFeatureProcessor.cpp]: removed :)

**Review by santorac — APPROVED** [review]

File: Gems/Atom/Feature/Common/Code/Include/Atom/Feature/Mesh/MeshFeatureProcessorInterface.h [hunk 2]
```diff
@@ -120,6 +120,8 @@ namespace AZ
             virtual bool GetRayTracingEnabled(const MeshHandle& meshHandle) const = 0;
             //! Sets the mesh as visible or hidden.  When the mesh is hidden it will not be rendered by the feature processor.
             virtual void SetVisible(const MeshHandle& meshHandle, bool visible) = 0;
+            //! Returns the visibility state of the mesh.
```
[the remark this answers is not in the document]
Comment (kh-huawei) [reply Gems/Atom/Feature/Common/Code/Include/Atom/Feature/Mesh/MeshFeatureProcessorInterface.h:123]: updated.

**Comment by hultonha** [comment]
Just kicked off a build 👍

**Comment by kh-huawei** [comment]
Rebased and added the GetVisible to the Mock MeshFeatureProcessor, so now the build should be successful

**Comment by rgba16f** [comment]
Started an AR run - https://jenkins.build.o3de.org/job/O3DE/view/change-requests/job/PR-10902/

**Pull request merged**